src/D2L.CodeStyle.TestAnalyzers/ServiceLocator/CustomServiceLocatorAnalyzerWhitelist.cs

```diff
@@ -0,0 +1,797 @@
+﻿using System.Collections.Immutable;
+
+namespace D2L.CodeStyle.TestAnalyzers.ServiceLocator {
+
+	internal static class CustomServiceLocatorAnalyzerWhitelist {
+
+		/// <summary>
+		/// A list of test fixtures or helper classes that already use custom
+		/// service locators.
+		/// </summary>
+		internal static readonly ImmutableHashSet<string> WhitelistedClasses = ImmutableHashSet.Create(

```

Comment:
Wow, really?

Reply:
Yeah :(. I've seen a few PRs from Patrick that remove a couple of these, but there are still a lot.